feat: trigger refresh through API endpoint

xunleii opened the pull request:
Add an API entrypoint to trigger manually the translation refresh

---
- chore(README): add missing code block
- chore(dependencies): upgrade all dependencies
- fix: use updated hashes on cron
- feat: add API entry to allow refresh from API call

xunleii committed:
chore(README): add missing code block

xunleii committed:
chore(dependencies): upgrade all dependencies

graillus commented on app.go:
```diff
@@ -76,7 +76,7 @@ func (app *App) Run() {
 		_, _ = cron.AddFunc(app.config.schedule, func() {
 			// make sure we have the latest translations
-			app.translations.Fetch()
+			hashes := app.translations.Fetch()
```
This is useless as the `hashes` variable is a pointer to the map containing the hashes. I guess it would be better to use a copy to avoid concurrent map access issues...

xunleii replied:
Here, the `hashes` is scoped to the anonymous `func` (I used `:=`) so we should not have any concurrences

xunleii replied:
But ok, I see. The returned `hashes` are a also stored into the refresher and we reuse this pointer again.

graillus replied:
Yes, but even if you define an other variable, you are still using the same map in memory because `app.translations.Fetch()` always returns a pointer to the same map instance. See https://github.com/graillus/translations-refresher/blob/main/loco.go#L36

graillus replied:
> Ok, I see. The returned `hashes` are a also stored into the refresher and we reuse this pointer again. There is a reason why we store the hash map internal instead of generating one every time we call `translations.Fetch` ?

I don't see any reason not to return a new instance

xunleii replied:
Because this is not a "fix", I removed the commit (useless and we can think that we've "fixed" something, that is not the case)

xunleii committed:
fix: range in Golang copy the value and not return a reference

In this case, the deployment will never been refreshed if the
annotations (hashes) didn't exists before calling Refresh

xunleii committed:
feat: add API entry to allow refresh from API call
